src/utils/ClockUtils.tsx

```diff
@@ -0,0 +1,72 @@
+// 현재 시간 포맷 (상단바)
+export const currentTime = () => {
+    return new Date().toLocaleTimeString('ko-KR', {
+        hour: '2-digit',
+        minute: '2-digit',
+        hour12: false,
+    });
+};
+
+// 타임스탬프 포맷
+export const formatTime = (date: any): string => {
+    // Date 객체가 아닌 경우
+    if (!(date instanceof Date) || isNaN(date.getTime())) {
+        return 'Invalid date';
+    }
+
+    const now = new Date();
+    const diffInTime = now.getTime() - date.getTime();
+    const diffInDays = Math.floor(diffInTime / (1000 * 60 * 60 * 24));
+
+    const hours = date.getHours();
+    const minutes = date.getMinutes();
+    const ampm = hours >= 12 ? '오후' : '오전';
+    const formattedHours = hours % 12 === 0 ? 12 : hours % 12;

```

Comment:
이거 아래처럼 쓰면 '오후','오전' 알아서 맞춰주더라구요!
```suggestion
     const formattedTime = new Date().toLocaleTimeString('ko-KR', {
        hour: '2-digit',
        minute: '2-digit',
      }),

```